spec/controllers/shops_controller_spec.rb

```diff
@@ -43,7 +43,8 @@
     get :index
 
     expect(response.body)
-      .to match(/"distributed_properties":\[{"id":\d+,"name":"certified","presentation":"certified"}\]/)
+      .to match(/"distributed_properties":\[{"id":\d+,"name":
+                "certified","presentation":"certified"}\]/x)
```

Comment:
TIL about "Free-spacing" mode in regex, provided by the `x` flag.

I think I would have instead simply `# rubocop:disable`d this one line. Or if writing from scratch perhaps use `hash_including()`, or just hardcode the ID in record.

Anyway this is fine :)

Reply:
Oh come on. Using `# rubocop: disable` is cheating, where is the fun in that?:)

Thanks for the heads up on `hash_including()`!

Reply:
😆